### PR TITLE
SVCPLAN-3691: Replace usage of org_asd group for access after ASD reorg

### LINKED ISSUE #5: Remove group org_asd
https://github.com/ncsa/puppet-profile_allow_ssh_from_bastion/blob/f8210141eac0c52e708fb8ad936e5da8f9d89eec/data/common.yaml#L14

### PR DESCRIPTION
**Pull request by billglick**
Fix #5 -- replacing org_asd with org_ici

### CODE REVIEW

File: data/common.yaml [hunk 1]
```diff
@@ -11,6 +11,6 @@ profile_allow_ssh_from_bastion::custom_cfg:
   PasswordAuthentication: "yes"
   PubkeyAuthentication: "no"
 profile_allow_ssh_from_bastion::groups:
-  - "org_asd"
+  - "org_ici"
```
Comment (bsper2) [inline data/common.yaml:14]: There a reason this module is getting `org_asd` replaced with `org_ici` and not `grp_asd_admin, org_cs, org_sp` like the others?

Comment (billglick) [reply data/common.yaml:14]: Yes. This module is not intended to be specific to 'ASD' usage. I do not think anything related to ASD should be left as a default.

I'm not sure what the default should be, but my gut would be to lean toward allowing all of ICI have access to our resources.

But we could also remove all org groups except org_irst if that makes more sense.

Comment (bsper2) [reply data/common.yaml:14]: Ah gotcha that makes sense.

**Review by bsper2 — APPROVED** [review]
Looks good

**Pull request merged**